setup.py

```diff
@@ -1,72 +1,69 @@
 # -*- coding: utf-8 -*-
```

Comment:
This file should be removed. Using the latest ndx-template, which I think you used here, this was replaced by `pyproject.toml`. The problem with this `setup.py` is that it copies the `spec` folder into `/src/pynwb/` on install, and so if you make changes to `/spec/ndx-microscopy.extensions.yaml`, you might expect the changes to be reflected in your environment, but no, the cached yaml in `/src/pynwb/spec/` is used instead.

Reply:
Makes sense - we'll deal with it in a follow-up since the way I copied this base didn't have the hatchling stuff in the TOML